lib/check_result.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 Security Scorecard Authors
```

Comment:
It is a standard approach to keep all the related structs in a single file instead of in separate files

https://golang.org/src/fmt/format.go

>All the files are based on features, and it is best to use a proximity principle approach, where you can find in the same file the definition of what you are using.
Generally, those features are grouped in one file per package, except for large ones, where one package is composed of many files (net, net/http)

https://stackoverflow.com/a/26229922/19407

Reply:
check_request is fine to be seperate, but i think check_result_utils can just go in this file.

Reply:
Done.